src/pages/blog-post.module.scss

```diff
@@ -0,0 +1,79 @@
+@import "/src/css/colors.scss";
+
+.container {
+  @media screen and (max-width: 768px) {
+    .headerContainer {
+      min-height: 100%;
+      padding: 5rem 1rem 8rem;
+    } 
+
+    .titleContainer {
+      margin: 2rem 0;
+    }
+
+    .postDateContainer {
+      display: flex;
```

Comment:
Please, only define the css that you want to apply on mobile and not on desktop, so avoid to define duplicate rules